common/reviews/api/heft-config-file.api.md

```diff
@@ -80,12 +83,23 @@ export interface IJsonPathsMetadata<TConfigurationFile> {
     [jsonPath: string]: IJsonPathMetadata<TConfigurationFile>;
 }
 
+// @beta
+export const InheritanceType: {
+    readonly append: "append";
+    readonly merge: "merge";
+    readonly replace: "replace";
+    readonly custom: "custom";
+};
+
 // @beta (undocumented)
-export enum InheritanceType {
-    append = "append",
-    custom = "custom",
-    merge = "merge",
-    replace = "replace"
+export type InheritanceType = (typeof InheritanceType)[keyof typeof InheritanceType];
+
+// @beta (undocumented)
```

Comment:
Should probably be documented

Reply:
This is documented, things just get weird due to declaration merging.